safe_cli/prompt_parser.py

```diff
@@ -418,6 +424,8 @@ def remove_delegate(args):
         parser.add_argument("token_address", type=check_ethereum_address)
         parser.add_argument("amount", type=int)
 
+    # Drain only needs destiny account
```

Comment:
If drain only needs `destiny account` why do you add it to the common parser arguments?